packages/react-proptypes/src/editorOptions.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    autoparagraph: PropTypes.bool,
+    placeholder: PropTypes.string,
+    formatting: PropTypes.shape({
+        strong: PropTypes.bool,
+        b: PropTypes.bool,
+        em: PropTypes.bool,
+        i: PropTypes.bool,
+        u: PropTypes.bool,
+        del: PropTypes.bool,
+        sub: PropTypes.bool,
+        sup: PropTypes.bool,
+        p: PropTypes.bool,
+        h1: PropTypes.bool,
+        h2: PropTypes.bool,
+        h3: PropTypes.bool,
+        h4: PropTypes.bool,
+        h5: PropTypes.bool,
+        h6: PropTypes.bool,
+        pre: PropTypes.bool,
+        removeFormat: PropTypes.bool,
+        table: PropTypes.bool,
+        a: PropTypes.bool,
+        ol: PropTypes.bool,
+        ul: PropTypes.bool,
+        right: PropTypes.bool,
```

Comment:
shouldn't we support the old schema where alignment and links and lists are placed outside of formatting? Or just break and ignore again?

Reply:
Good question, this is just documenting the status quo that currently exists in code. I think we have to decide if we say a) yes we want to have this and document it now or b) we go back say only use aloha config for now because we are still looking at how the new config might look like, then discuss and finalize it so it is stable and then document it when it gets implemented that way. I am fine with either but we need to decide that.

Reply:
Again, might be solved with the current code already!?